google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -3235,6 +3235,85 @@ public void testEnableAndDisableUniformBucketLevelAccessOnExistingBucket() throw
     }
   }
 
+  @Test
+  public void testUnspecifiedAndEnforcedPublicAccessPreventionOnBucket() throws Exception {
+    String papBucket = RemoteStorageHelper.generateBucketName();
+    try {
+      BucketInfo.IamConfiguration iamConfiguration =
+          BucketInfo.IamConfiguration.newBuilder()
+              .setIsUniformBucketLevelAccessEnabled(true)
+              .build();
+      Bucket bucket =
+          storage.create(
+              Bucket.newBuilder(papBucket).setIamConfiguration(iamConfiguration).build());
+      assertNull(bucket.getIamConfiguration().getPublicAccessPrevention());
```

Comment:
publicAccessPrevention should never be null

Reply:
This looks like a bug, will follow-up!

Reply:
Blocked for now as this is a backend bug as discussed offline. Thanks @shaffeeullah and @tritone

Reply:
@frankyn Can you please remove this check in the integration test? Public access prevention should not be null, but also I think this is validating the API functionality rather than the client library functionality. I don't think we should be checking the public access prevention default.

Reply:
Gotcha, so just validate that public data is not accessible when enabled and is accessible when disabled. Will follow-up